vault-validation/pom.xml

```diff
@@ -124,6 +124,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
```

Comment:
I don't see it being used in this module....

Reply:
Not having it here causes a test failure.